lib/xcov-core.rb

```diff
@@ -1,6 +1,7 @@
 require 'xcov-core/version'
 require 'xcov'
 require 'json'
+require 'fileutils'
 require 'tempfile'
```

Comment:
I still use the tempfile gem on line 18. I just give it the folder to create the temp file in instead of letting it choose. that way we always know we have write access